### ISSUE TITLE
PluginView layout is overlapped with PluginSourceBrowser

### ISSUE THREAD

**Issue by kajornsakp** [matryer/xbar#658]
Hi! I just downloaded xbar v2.0.32-beta from latest release and seems like PluginView layout is broken on my machine. `PluginSourceBrowser` is hover the rest of PluginView area and I can't tap the install button 😂 .

Some details about my machine
- MacBook Pro (15-inch, 2018)
- macOS Big Sur 11.1
- xbar v2.0.32-beta

<img width="1792" alt="Screen Shot 2021-03-22 at 13 40 20" src="https://user-images.githubusercontent.com/10228783/111950278-a04deb80-8b14-11eb-91d4-b41de49fadf3.png">

**Comment by matryer** [comment]
Hi @kajornsakp have you tried frisbeeing your laptop into the sea?

**Comment by akexorcist** [comment]
@matryer Strongly agree. He should try to do that.

**Comment by matryer** [comment]
But seriously, this bug gets fixed if you upgrade Big Sur to the latest. It was happening in Catalina too, but we fixed it.

@leaanthony What was the fix you did?

**Comment by leaanthony** [comment]
It was a CSS rule. It made no sense at the time, but does it ever? There's a one line commit in the old repo as it happened before merge.

**Comment by matryer** [comment]
@kajornsakp Does this consistently happen for you? Does the Refresh button help at all?

**Comment by kajornsakp** [comment]
@matryer yes, refresh button didn't help. Even force quit and restart the app still give the same result

**Comment by matryer** [comment]
@kajornsakp I know upgrading to the latest Big Sur will fix this, but I'd love to try and figure out how to fix it without forcing people to do that.

**Comment by leaanthony** [comment]
@kajornsakp What version of Safari you have installed? Thanks.

**Comment by leaanthony** [comment]
This was the fix for Catalina:
![image](https://user-images.githubusercontent.com/1943904/112616957-7419ce00-8e78-11eb-9e8d-c8293e87052f.png)

**Comment by kajornsakp** [comment]
@leaanthony Safari version : Version 14.0.2 (16610.3.7.1.9)

@matryer Yeah, I wish I could update to latest version of Big Sur too. But this mac is controlled by MDM service from my company 😅

**Comment by leaanthony** [comment]
Thanks. I'm wondering if we can get an archive version of safari at that version

**Comment by leaanthony** [comment]
This should either be fixed by now (depending on OS version) or will be fixed by #705
